### PR TITLE
Update resource files to support localization

### PR DESCRIPTION
**Pull request by dhagens**
@jmccausland, please review and merge

### CODE REVIEW

File: source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml [hunk 1]
```diff
@@ -22,10 +23,11 @@
             <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <GroupBox Header="Input">
+        <GroupBox Header="{x:Static prop:Resources.HeaderInput}">
             <Grid>
                 <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="*" />
+                    <ColumnDefinition Width="5*" />
```
Comment (jmccausland) [inline source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml]: What is the justification for changing the column definitions?

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 2]
```diff
@@ -42,7 +42,7 @@ public CoordinateConversionViewModel()
         public static CoordinateConversionLibraryConfig AddInConfig = new CoordinateConversionLibraryConfig();
 
         // InputCoordinate
-        private string inputCoordinate = "70.49N40.32W";
+        private string inputCoordinate = Properties.Resources.StringInputCoord;
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: remove from resource file

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 3]
```diff
@@ -85,21 +85,21 @@ private void UpdateOutputs()
                             var splits = output.Format.Split(new char[] { 'X' }, StringSplitOptions.RemoveEmptyEntries);
                             if (splits.Count() == 2)
                             {
-                                props.Add("Lat", cdd.ToString(splits[0].Trim(), new CoordinateDDFormatter()));
-                                props.Add("Lon", cdd.ToString("X" + splits[1].Trim(), new CoordinateDDFormatter()));
+                                props.Add(Properties.Resources.StringLat, cdd.ToString(splits[0].Trim(), new CoordinateDDFormatter()));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: localizing these may cause issues with the coordinate properties layout

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 4]
```diff
@@ -85,21 +85,21 @@ private void UpdateOutputs()
...
+                                props.Add(Properties.Resources.StringLon, cdd.ToString(Properties.Resources.StringX + splits[1].Trim(), new CoordinateDDFormatter()));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: "X" is a non-localizable formatting character, remove

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 5]
```diff
@@ -85,21 +85,21 @@ private void UpdateOutputs()
...
                             }
                             else
                             {
                                 splits = output.Format.Split(new char[] { 'Y' }, StringSplitOptions.RemoveEmptyEntries);
                                 if (splits.Count() == 2)
                                 {
-                                    props.Add("Lon", cdd.ToString(splits[0].Trim(), new CoordinateDDFormatter()));
-                                    props.Add("Lat", cdd.ToString("Y" + splits[1].Trim(), new CoordinateDDFormatter()));
+                                    props.Add(Properties.Resources.StringLon, cdd.ToString(splits[0].Trim(), new CoordinateDDFormatter()));
+                                    props.Add(Properties.Resources.StringLat, cdd.ToString(Properties.Resources.StringY + splits[1].Trim(), new CoordinateDDFormatter()));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: "Y" is a non-localizable formatting character, remove

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 6]
```diff
@@ -114,21 +114,21 @@ private void UpdateOutputs()
                             var splits = output.Format.Split(new char[] { 'X' }, StringSplitOptions.RemoveEmptyEntries);
                             if (splits.Count() == 2)
                             {
-                                props.Add("Lat", cdms.ToString(splits[0].Trim(), new CoordinateDMSFormatter()));
-                                props.Add("Lon", cdms.ToString("X" + splits[1].Trim(), new CoordinateDMSFormatter()));
+                                props.Add(Properties.Resources.StringLat, cdms.ToString(splits[0].Trim(), new CoordinateDMSFormatter()));
+                                props.Add(Properties.Resources.StringLon, cdms.ToString(Properties.Resources.StringX + splits[1].Trim(), new CoordinateDMSFormatter()));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: "X" is a non-localizable formatting character, remove

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 7]
```diff
@@ -114,21 +114,21 @@ private void UpdateOutputs()
...
                             }
                             else
                             {
                                 splits = output.Format.Split(new char[] { 'Y' }, StringSplitOptions.RemoveEmptyEntries);
                                 if (splits.Count() == 2)
                                 {
-                                    props.Add("Lon", cdms.ToString(splits[0].Trim(), new CoordinateDMSFormatter()));
-                                    props.Add("Lat", cdms.ToString("Y" + splits[1].Trim(), new CoordinateDMSFormatter()));
+                                    props.Add(Properties.Resources.StringLon, cdms.ToString(splits[0].Trim(), new CoordinateDMSFormatter()));
+                                    props.Add(Properties.Resources.StringLat, cdms.ToString(Properties.Resources.StringY + splits[1].Trim(), new CoordinateDMSFormatter()));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: "Y" is a non-localizable formatting character, remove

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 8]
```diff
@@ -143,21 +143,21 @@ private void UpdateOutputs()
                             var splits = output.Format.Split(new char[] { 'X' }, StringSplitOptions.RemoveEmptyEntries);
                             if (splits.Count() == 2)
                             {
-                                props.Add("Lat", ddm.ToString(splits[0].Trim(), new CoordinateDDMFormatter()));
-                                props.Add("Lon", ddm.ToString("X" + splits[1].Trim(), new CoordinateDDMFormatter()));
+                                props.Add(Properties.Resources.StringLat, ddm.ToString(splits[0].Trim(), new CoordinateDDMFormatter()));
+                                props.Add(Properties.Resources.StringLon, ddm.ToString(Properties.Resources.StringX + splits[1].Trim(), new CoordinateDDMFormatter()));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: "X" is a non-localizable formatting character, remove

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 9]
```diff
@@ -143,21 +143,21 @@ private void UpdateOutputs()
...
                             }
                             else
                             {
                                 splits = output.Format.Split(new char[] { 'Y' }, StringSplitOptions.RemoveEmptyEntries);
                                 if (splits.Count() == 2)
                                 {
-                                    props.Add("Lon", ddm.ToString(splits[0].Trim(), new CoordinateDDMFormatter()));
-                                    props.Add("Lat", ddm.ToString("Y" + splits[1].Trim(), new CoordinateDDMFormatter()));
+                                    props.Add(Properties.Resources.StringLon, ddm.ToString(splits[0].Trim(), new CoordinateDDMFormatter()));
+                                    props.Add(Properties.Resources.StringLat, ddm.ToString(Properties.Resources.StringY + splits[1].Trim(), new CoordinateDDMFormatter()));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: "Y" is a non-localizable formatting character, remove

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 10]
```diff
@@ -182,10 +182,10 @@ private void UpdateOutputs()
                             CoordinateMGRS.TryParse(coord, out mgrs))
                         {
                             output.OutputCoordinate = mgrs.ToString(output.Format, new CoordinateMGRSFormatter());
-                            props.Add("GZD", mgrs.GZD);
-                            props.Add("Grid Sq", mgrs.GS);
-                            props.Add("Easting", mgrs.Easting.ToString("00000"));
-                            props.Add("Northing", mgrs.Northing.ToString("00000"));
+                            props.Add(Properties.Resources.StringGZD, mgrs.GZD);
+                            props.Add(Properties.Resources.StringGridSq, mgrs.GS);
+                            props.Add(Properties.Resources.StringEasting, mgrs.Easting.ToString(Properties.Resources.StringZeros));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: this .ToString() should should not be localized

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 11]
```diff
@@ -182,10 +182,10 @@ private void UpdateOutputs()
...
+                            props.Add(Properties.Resources.StringNorthing, mgrs.Northing.ToString(Properties.Resources.StringZeros));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: this .ToString() should should not be localized

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 12]
```diff
@@ -195,10 +195,10 @@ private void UpdateOutputs()
                             CoordinateUSNG.TryParse(coord, out usng))
                         {
                             output.OutputCoordinate = usng.ToString(output.Format, new CoordinateMGRSFormatter());
-                            props.Add("GZD", usng.GZD);
-                            props.Add("Grid Sq", usng.GS);
-                            props.Add("Easting", usng.Easting.ToString("00000"));
-                            props.Add("Northing", usng.Northing.ToString("00000"));
+                            props.Add(Properties.Resources.StringGZD, usng.GZD);
+                            props.Add(Properties.Resources.StringGridSq, usng.GS);
+                            props.Add(Properties.Resources.StringEasting, usng.Easting.ToString(Properties.Resources.StringZeros));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: this .ToString() should should not be localized

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 13]
```diff
@@ -195,10 +195,10 @@ private void UpdateOutputs()
...
+                            props.Add(Properties.Resources.StringNorthing, usng.Northing.ToString(Properties.Resources.StringZeros));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: this .ToString() should should not be localized

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 14]
```diff
@@ -208,9 +208,9 @@ private void UpdateOutputs()
                             CoordinateUTM.TryParse(coord, out utm))
                         {
                             output.OutputCoordinate = utm.ToString(output.Format, new CoordinateUTMFormatter());
-                            props.Add("Zone", utm.Zone.ToString() + utm.Hemi);
-                            props.Add("Easting", utm.Easting.ToString("000000"));
-                            props.Add("Northing", utm.Northing.ToString("0000000"));
+                            props.Add(Properties.Resources.StringZone, utm.Zone.ToString() + utm.Hemi);
+                            props.Add(Properties.Resources.StringEasting, utm.Easting.ToString(Properties.Resources.StringZeros));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: this .ToString() should should not be localized

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 15]
```diff
@@ -208,9 +208,9 @@ private void UpdateOutputs()
...
+                            props.Add(Properties.Resources.StringNorthing, utm.Northing.ToString(Properties.Resources.StringZeros));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: this .ToString() should should not be localized

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/EditOutputCoordinateViewModel.cs [hunk 16]
```diff
@@ -29,15 +29,34 @@ public EditOutputCoordinateViewModel()
...
+                                                                    Properties.Resources.CategoryListDMS,
+                                                                    Properties.Resources.CategoryListGARS,
+                                                                    Properties.Resources.CategoryListMGRS,
+                                                                    Properties.Resources.CategoryListUSNG,
+                                                                    Properties.Resources.CategoryListUTM };
             }
-            FormatList = new ObservableCollection<string>() { "One", "Two", "Three", "Four", "Five", "Six", "Seven", "Custom" };
-            Sample = "Sample";
-            Format = "Y-+0.####,X-+0.####";
+            FormatList = new ObservableCollection<string>() { Properties.Resources.FormatListOne,
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/EditOutputCoordinateViewModel.cs]: this is test data, do not localize

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 3 above)
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: i double checked this, it'll be good to go, I'll remove this one

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 15 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 14 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 13 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 12 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 11 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 10 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 9 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 8 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 7 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 6 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 5 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 4 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 2 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

**Comment by dhagens** [comment]
@jmccausland, changes have been made

File: source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml [hunk 17]
```diff
@@ -22,10 +23,11 @@
             <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <GroupBox Header="Input">
+        <GroupBox Header="{x:Static prop:Resources.HeaderInput}">
             <Grid>
                 <Grid.ColumnDefinitions>
                     <ColumnDefinition Width="*" />
+                    <ColumnDefinition Width="*"/>
```
Comment (jmccausland) [inline source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml]: remove

File: source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml [hunk 18]
```diff
@@ -41,7 +43,7 @@
                           IsEditable="True"
                           ItemsSource="{Binding InputCoordinateHistoryList}"
                           Text="{Binding InputCoordinate,
-                                         UpdateSourceTrigger=PropertyChanged}">
+                                         UpdateSourceTrigger=PropertyChanged}" Grid.ColumnSpan="2">
```
Comment (jmccausland) [inline source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml]: undo change

File: source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml [hunk 19]
```diff
@@ -58,39 +60,40 @@
                         <KeyBinding Key="Enter" Command="{Binding FlashPointCommand}" />
                     </ComboBox.InputBindings>
                 </ComboBox>
-                <Button Grid.Column="1"
+                <Button Grid.Column="2"
```
Comment (jmccausland) [inline source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml]: undo change

File: source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml [hunk 20]
```diff
@@ -58,39 +60,40 @@
...
                         Margin="3,3,0,0"
+                        ToolTip="{x:Static libprop:Resources.TooltipEditProperties}"
                         Style="{StaticResource buttonProperties}" />
-                <Button Grid.Column="2"
+                <Button Grid.Column="3"
```
Comment (jmccausland) [inline source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml]: undo change

File: source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml [hunk 21]
```diff
@@ -58,39 +60,40 @@
...
                         Command="{Binding AddNewOCCommand}"
-                        ToolTip="Add"
+                        ToolTip="{x:Static libprop:Resources.TooltipAdd}"
                         Style="{StaticResource BorderedButtonStyle}">
                     <Image Width="18"
                            Height="18"
                            Source="/CoordinateConversionLibrary;component/Images/hand.png" />
                 </Button>
-                <ToggleButton Grid.Column="3"
+                <ToggleButton Grid.Column="4"
```
Comment (jmccausland) [inline source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml]: undo change

File: source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml [hunk 22]
```diff
@@ -58,39 +60,40 @@
...
                               IsChecked="{Binding IsToolActive}"
-                              ToolTip="Map Point Tool"
+                              ToolTip="{x:Static libprop:Resources.TooltipMapPointTool}"
                               Style="{StaticResource BorderedTButtonStyle}">
                     <Image Width="18"
                            Height="18"
                            Source="/CoordinateConversionLibrary;component/Images/Add_Point.png" />
                 </ToggleButton>
-                <Button Grid.Column="4"
+                <Button Grid.Column="5"
```
Comment (jmccausland) [inline source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml]: undo change

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 2 above)
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: undo change

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 5 above)
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: undo change, put the "Y" back

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 6 above)
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: undo change, put the "X" back

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 7 above)
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: undo change, put the "Y" back

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 8 above)
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: undo change, put the "X" back

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 9 above)
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: again, undo change, put the "Y" back

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 11 above)
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: undo change, nothing has changed, are you pushing your commits?  do not edit .ToString()

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 12 above)
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: undo change

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 13 above)
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: undo change

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 14 above)
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: undo change

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 15 above)
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: undo change

File: source/CoordinateConversion/CoordinateConversionLibrary/Properties/Resources.resx [hunk 23]
```diff
@@ -147,13 +168,85 @@
...
+    <value>Easting</value>
+  </data>
+  <data name="StringGridSq" xml:space="preserve">
+    <value>Grid Sq</value>
+  </data>
+  <data name="StringGZD" xml:space="preserve">
+    <value>GZD</value>
+  </data>
+  <data name="StringInputCoord" xml:space="preserve">
+    <value>70.49N40.32W</value>
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/Properties/Resources.resx]: remove from resource file

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 24]
```diff
@@ -208,9 +208,9 @@ private void UpdateOutputs()
                             CoordinateUTM.TryParse(coord, out utm))
                         {
                             output.OutputCoordinate = utm.ToString(output.Format, new CoordinateUTMFormatter());
-                            props.Add("Zone", utm.Zone.ToString() + utm.Hemi);
-                            props.Add("Easting", utm.Easting.ToString("000000"));
-                            props.Add("Northing", utm.Northing.ToString("0000000"));
+                            props.Add(Properties.Resources.StringZone, utm.Zone.ToString() + utm.Hemi);
+                            props.Add(Properties.Resources.StringEasting, utm.Easting.ToString("00000"));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: the number of zeros is very important, undo change

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs [hunk 25]
```diff
@@ -208,9 +208,9 @@ private void UpdateOutputs()
...
+                            props.Add(Properties.Resources.StringNorthing, utm.Northing.ToString("00000"));
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: the number of zeros is very important

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/EditOutputCoordinateViewModel.cs [hunk 26]
```diff
@@ -25,17 +25,37 @@ namespace CoordinateConversionLibrary.ViewModels
 {
     public class EditOutputCoordinateViewModel : BaseViewModel
     {
+        private const string CustomString = "Custom";
```
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/EditOutputCoordinateViewModel.cs]: This must be in the resource file

**Comment by jmccausland** [comment]
@dhagens there is still changes that need to be undone before acceptance

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 25 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 24 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Fixed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 2 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Removed

File: source/CoordinateConversion/CoordinateConversionLibrary/Properties/Resources.resx (hunk 23 above)
Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/Properties/Resources.resx]: Removed

File: source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml (hunk 18 above)
Comment (jmccausland) [inline source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml]: get rid of column span

File: source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml (hunk 17 above)
Comment (dhagens) [inline source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml]: Fixed

File: source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml (hunk 19 above)
Comment (dhagens) [inline source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml]: Fixed

File: source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml (hunk 18 above)
Comment (dhagens) [inline source/CoordinateConversion/ArcMapAddinCoordinateConversion/DockableWindowCoordinateConversion.xaml]: Removed

File: source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs (hunk 2 above)
Comment (jmccausland) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: this is still not back to the original line

private string inputCoordinate = "70.49N40.32W";

Comment (dhagens) [inline source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/CoordinateConversionViewModel.cs]: Updated

**Pull request merged**